11-kerberos-ticket-policy.rst

```diff
@@ -0,0 +1,387 @@
+Unit 11: Kerberos ticket policy
+=========================================
+
+**Prerequisites:**
+
+- `Unit 3: User management and Kerberos authentication <3-user-management.rst>`_
+
+In this module you will explore how to manage properties of Kerberos
+authentication and authorization in FreeIPA.
+
+Kerberos ticket policies
+------------------------
+
+FreeIPA's primary authentication mechanism is based on Kerberos infrastructure.
+Each user has associated Kerberos principal and potential aliases. Each FreeIPA
+service has own Kerberos service and, optionally, alias names as well. Depending
+on who initiates a communication flow, in Kerberos terminology there are client
+and a target (server) principals. Any Kerberos principal can be used in both
+client and server roles, though typically FreeIPA services are used as targets
+and users are used as clients.
+
+A client first authenticates to a Key Distribution Center and obtains a ticket
+granting ticket (TGT). With the ticket granting ticket in possession, the client
+then asks a KDC for a service ticket to the target Kerberos principal. After the
+service ticket was issued by the KDC, the client can initiate its
+application-specific communication to the target application server.
+
+When a KDC issues a ticket, there are few properties that can be controlled with
+the help of a Kerberos policy in FreeIPA. Each ticket has own life time and a
+potential renewal age: a ticket can be renewed before its life time has ended
+but until the renewal age expired.
+
+The combination of Kerberos ticket life time and renewal age altogether
+comprises a **Kerberos ticket policy**. The policy itself is not stored in a single
+place because individual parts of it are applied to different objects and at
+different stages of authentication and authorization processes. Instead, it is
+split into several parts which are associated with the corresponding Kerberos
+principals, whether they are used as clients or as targets.
+
+For users, Kerberos ticket life time and renewal age can be managed with
+Kerberos ticket policy commands described in ``ipa help krbtpolicy`` manual.
+
+If no specific policy is associated with a user, a default one is applied. To
+manage default policy the same ``ipa krbtpolicy-*`` commands are used, without
+explicit user name.
+
+Display the current Kerberos ticket policy::
+
+  [admin@client]$ kinit admin
+  Password for admin@IPADEMO.LOCAL:
+  [admin@client]$ ipa krbtpolicy-show
+    Max life: 86400
+    Max renew: 604800
+
+Reset the policy to the default::
```

Comment:
This resets the default policy to the default :-)

Maybe "installation default"?